File: app/src/main/java/dev/ragnarok/fenrir/settings/SwipesChatMode.java
package dev.ragnarok.fenrir.settings;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@IntDef({SwipesChatMode.DISABLED, SwipesChatMode.SLIDR})
@Retention(RetentionPolicy.SOURCE)
public @interface SwipesChatMode {
    int DISABLED = 0;
    int SLIDR = 1;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/settings/SwipesChatMode.kt
package dev.ragnarok.fenrir.settings

import androidx.annotation.IntDef

@IntDef(SwipesChatMode.DISABLED, SwipesChatMode.SLIDR)
@Retention(AnnotationRetention.SOURCE)
annotation class SwipesChatMode {
    companion object {
        const val DISABLED = 0
        const val SLIDR = 1
    }
}